.github/workflows/run_system_tests.yml

```diff
@@ -0,0 +1,21 @@
+name: run_system_tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  run_system_tests:
+    runs-on:
+      - self-hosted
+      - rdss-nidaqmxbot-${{ matrix.configuration }}
+    timeout-minutes: 80
+    strategy:
+      matrix:
+        configuration: [win-10-py32, win-10-py64]
+    steps:
+      - name: checkout repository
+        uses: actions/checkout@v3
+      - name: Install dependencies
```

Comment:
This should also run `nidaqmxconfig` to import the MAX config. That way we don't need to rebuild the AMI when the MAX config changes.

You will need to update the AMI setup to add `nidaqmxconfig` to the system path, then come back to this in a separate PR.

Reply:
@bkeryan I have created a **[Technical Debt 2436601](https://dev.azure.com/ni/DevCentral/_workitems/edit/2436601): Import nidaqmxconfig in GitHub workflow** to track this item.